src/evm_transition_tool/transition_tool.py

```diff
@@ -311,6 +318,18 @@ class TransitionToolData:
         fork_name: str
         chain_id: int = field(default=1)
         reward: int = field(default=0)
+        empty_string_to: bool = field(default=False)
+
+        def __post_init__(self):
+            """
+            Ensure that the `to` field of transactions is not None
+            """
+            if self.empty_string_to:
+                for tx in self.txs:
+                    if "to" not in tx:
+                        tx["to"] = ""
+                    elif tx["to"] is None:
+                        tx["to"] = ""
```

Comment:
I think we can do this:
```suggestion
                for tx in self.txs:
                    tx["to"] = tx.get("to") or ""
```